packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -387,6 +387,7 @@ export type CallCompositeIcons = {
     PeoplePaneAddPerson?: JSX.Element;
     PeoplePaneOpenDialpad?: JSX.Element;
     DialpadStartCall?: JSX.Element;
+    InvalidRoom?: JSX.Element;
```

Comment:
For consistency, can we use `NoticePageInvalidRoom` -- don't love this naming but just to keep it consistent with the other end call page icons